src/locale/defaults.h

```diff
@@ -207,6 +213,18 @@
 #ifndef D_STR_SENSOR
 #define D_STR_SENSOR "Sensor"
 #endif  // D_STR_SENSOR
+#ifndef D_STR_ABSENSEDETECT
+#define D_STR_ABSENSEDETECT "Absense detect"
+#endif  // D_STR_ABSENSEDETECT
+#ifndef D_STR_DIRECTINDIRECTMODE
+#define D_STR_DIRECTINDIRECTMODE "Direct / Indirect mode"
+#endif  // D_STR_DIRECTINDIRECTMODE
+#ifndef D_STR_DIRECT
+#define D_STR_DIRECT "Direct"
+#endif  // D_STR_DIRECT
+#ifndef D_STR_INDIRECT
+#define D_STR_INDIRECT "Indirect"
+#endif  // D_STR_INDIRECT
```

Comment:
If we re-order them, we can use the sub-words for the compound term. This can help for internationalisation/translations.
```c++
#ifndef D_STR_DIRECT
#define D_STR_DIRECT "Direct"
#endif  // D_STR_DIRECT
#ifndef D_STR_INDIRECT
#define D_STR_INDIRECT "Indirect"
#endif  // D_STR_INDIRECT
#ifndef D_STR_DIRECTINDIRECTMODE
#define D_STR_DIRECTINDIRECTMODE  D_STR_DIRECT" / " D_STR_INDIRECT " mode"
#endif  // D_STR_DIRECTINDIRECTMODE
```